src/OSVRTrackedDevice.cpp

```diff
@@ -848,21 +848,34 @@ std::string OSVRTrackedDevice::GetStringTrackedDeviceProperty(vr::ETrackedDevice
     return default_value;
 }
 
-void OSVRTrackedDevice::HmdTrackerCallback(void* userdata, const OSVR_TimeValue*, const OSVR_PoseReport* report)
+void OSVRTrackedDevice::HmdTrackerCallback(void* userdata, const OSVR_TimeValue* timeval, const OSVR_PoseReport* report)
 {
     if (!userdata)
         return;
 
     auto* self = static_cast<OSVRTrackedDevice*>(userdata);
 
+    // Get angular velocity in correct format for SteamVR
+    OSVR_TimeValue timeval2;
+    OSVR_AngularVelocityState state;
+    OSVR_VelocityState velocitystate;
+
+    osvrGetVelocityState(self->trackerInterface_.get(), &timeval2, &velocitystate);
+
+    double dt = state.dt;
+    Eigen::Quaterniond poserotation = osvr::util::fromQuat(report->pose.rotation);
+    Eigen::Quaterniond angvel_incrementalRotation = osvr::util::fromQuat(velocitystate.angularVelocity.incrementalRotation);
+    angvel_incrementalRotation = poserotation.inverse() * angvel_incrementalRotation * poserotation;
```

Comment:
What is the intention behind this line? Wouldn't the `poserotation.inverse()` pre-rotation be eliminated by the `poserotation` post-rotation?